run/types/DeviceWrapper.ts

```diff
@@ -449,38 +465,43 @@ export class DeviceWrapper implements SharedDeviceInterface {
   public async grabTextFromAccessibilityId(
     accessibilityId: AccessibilityId
   ): Promise<string> {
-    const elementId = await this.waitForElementToBePresent({
+    const elementId = await this.waitForTextElementToBePresent({
       strategy: "accessibility id",
       selector: accessibilityId,
     });
-
     const text = await this.getTextFromElement(elementId);
     return text;
   }
 
-  public async deleteTextAndroid(accessibilityId: AccessibilityId) {
+  public async deleteText(accessibilityId: AccessibilityId) {
     const el = await this.findElementByAccessibilityId(accessibilityId);
-    await this.longClick(el, 200);
-
+    await this.longClick(el, 500);
+    if (this.isIOS()) {
+      await this.clickOnElementByText({
+        strategy: "id",
+        selector: "Select All",
+        text: "Select All",
+      });
+    }
```

Comment:
I am not too sure what is this doing

Reply:
clear isn't working as intended on ios and this is the only way to replace text